### PR TITLE
misc updates for Name Request

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* https://github.com/bcgov/entity/issues/5471

*Description of changes:*
- updated step 2 NR image
- added config key for new Name Request URL
- updated Name Request Button to use new/old URL according to FF
- updated Request Name View to use new/old URLs according to FF
- updated session vars in unit tests (for possible future use)
- added entries to mock to increase code coverage

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/public/config/configuration.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "BCEID_OSD_LINK": "https://www.test.bceid.ca/os/?7686&SkipTo=Basic",
   "FILE_SERVER_URL":"http://minio-1rdehl-dev.pathfinder.gov.bc.ca/public",
   "NRO_URL": "https://dev.bcregistrynames.gov.bc.ca/nro/",
+  "NAME_REQUEST_URL": "https://dev.bcregistry.ca/namerequest/",
```
Comment (severinbeauvais) [inline auth-web/public/config/configuration.json:15]: I also added this to OS for Dev/Test/Prod.

File: auth-web/src/views/auth/home/RequestNameView.vue [hunk 2]
```diff
@@ -60,16 +62,23 @@ import NumberedCompanyTooltip from '@/components/auth/common/NumberedCompanyTool
...
     { text: 'Create a unique name that ensures the public is not confused or misled by similar corporate names.' },
     { text: 'Submit this name for examination by the Business Registry.' },
     { text: 'If your name is approved, you can use it to incorporate or register your business.' }
   ]
 
-  private get nameRequestStatusUrl () {
-    return `${this.nroUrl}nro.htm?_flowId=anonymous-monitor-flow&_flowExecutionKey=e1s1`
+  private get nameRequestExistingUrl () {
+    return LaunchDarklyService.getFlag(LDFlags.LinkToNewNameRequestApp)
+      ? `${ConfigHelper.getNameRequestUrl()}existing`
```
Comment (severinbeauvais) [inline auth-web/src/views/auth/home/RequestNameView.vue:74]: I am assuming that, at some point in the future, this URL will open the "Existing Name Request" tab. In the meantime, NR UI has a fallback route so this works.

Comment (cameron-freshworks) [reply auth-web/src/views/auth/home/RequestNameView.vue:74]: interesting!

Comment (severinbeauvais) [reply auth-web/src/views/auth/home/RequestNameView.vue:74]: Yep.

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

**Review by nitheesh-aot — APPROVED** [review]

**Pull request merged**